### PR TITLE
Removing Graph Builders as user facing class

### PR DESCRIPTION
**Pull request by miratepuffin**

### CODE REVIEW

File: core/src/main/scala/com/raphtory/Raphtory.scala [hunk 1]
```diff
@@ -36,7 +38,7 @@ import scala.collection.mutable.ArrayBuffer
   * graph.deployment.stop()
   * }}}
   *
-  * @see [[api.input.GraphBuilder GraphBuilder]]
+  * @see [[GraphBuilder GraphBuilder]]
```
Comment (miratepuffin) [inline core/src/main/scala/com/raphtory/Raphtory.scala:41]: Change to graph

File: core/src/main/scala/com/raphtory/Raphtory.scala [hunk 2]
```diff
@@ -7,12 +7,14 @@ import com.raphtory.internals.context.LocalContext
...
 import com.raphtory.internals.context.RemoteContext
 import com.raphtory.internals.context.LocalContext.createName
+import com.raphtory.internals.graph.GraphBuilder
 import com.raphtory.internals.management._
 import com.raphtory.internals.management.id.IDManager
 import com.raphtory.internals.management.id.LocalIDManager
 import com.raphtory.internals.management.id.ZooKeeperCounter
 import com.raphtory.internals.management.id.ZookeeperLimitedPool
 import com.typesafe.config.Config
+import net.openhft.hashing.LongHashFunction
```
Comment (miratepuffin) [inline core/src/main/scala/com/raphtory/Raphtory.scala:17]: Remove

**Review by miratepuffin — COMMENTED** [review]
Make concrete add/remove functions inside graph trait which calls handleUpdate
handle update to be implemented in graph builder and query sending etc.
add abstract graph index

**Comment by Haaroon** [comment]
i have tested the python variant of this and it works, the sample.py works fine, it creates graphs and obtains results

**Review by Haaroon — APPROVED** [review]

**Pull request merged**